### ISSUE TITLE
compiling

### ISSUE THREAD

**Issue by coachshea** [graywolf/gbook#2]
You had helped me once before when I was having trouble compiling gbook. You got it working and it has been great ever since. I am now compiling it on a different computer. I thought that I had learned enough from the experience that I could do it without having to bother you, but I am having a different problem this time. I am able to run the cmake .. command and it appears to run fine. When I then run make I get the following error:

    .../gbook/include/oauth2.h:6:23: fatal error: json/json.h: No such file or directory

**Comment by graywolf** [comment]
Hi :) I'm glad you like gbook.

Now to your problem, could you please verify that you have JSONcpp library installed? I don't know what distro are you using, on archlinux it's a package `jsoncpp`. If you have distro which splits stuff into normal and development packages, make sure both are installed (I don't really remember if that's the case with jsoncpp).

Let me know if installing it helped :)

**Comment by coachshea** [comment]
Sorry it took me so long to respond. Life took some unexpected and busy turns. So, I'm on a debian stretch. I had installed the jsoncpp library, but for some reason it didn't seem to be placed in the include path. I was under the impression (I apologize if this is my ignorance of cpp programming) that when headers where included in quotes they were usually local headers and brackets meant system headers. Just be safe (I believe safe) I added a json directory in my copy of cbatticon and copied all of the jsoncpp header files into it. It then compiled fine and seems to be working perfectly. Thank you for your help.
